freqtrade/tests/strategy/test_strategy.py

```diff
@@ -69,8 +69,8 @@ def test_load_strategy(result):
 
 
 def test_load_strategy_base64(result):
-    with open("freqtrade/tests/strategy/test_strategy.py", "r") as file:
-        encoded_string = urlsafe_b64encode(file.read().encode("utf-8")).decode("utf-8")
+    with open("freqtrade/tests/strategy/test_strategy.py", "rb") as file:
```

Comment:
Are you sure about "rb" mode? This may produce (not sure, do not remember right now how base64 behaves here) diff. results for Linux/MacOS and Windows after base64-encoding (due to the differences in CRLF)...

Reply:
good point, I'll test it on windows tomorrow (not concerned about linux/mac difference, these should be fine.

However, as we used to `read().encode("utf-8)` (normally you `decode()`) it's a very oddly written test anyway

Reply:
There's no problem with tests on windows (neither on WSL, nor in "real" windows mode).